### PR TITLE
profiles/updates: Rename dev-java/{ant-ivy -> apache-ivy}

### PR DESCRIPTION
**Pull request by vaukai**
---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

**Review by fordfrog — APPROVED** [review]

**Review by fordfrog — CHANGES_REQUESTED** [review]
package move also needs info in `profiles/updates`: https://devmanual.gentoo.org/ebuild-maintenance/package-moves/index.html#moving-or-renaming-a-package

**Comment by vaukai** [comment]
> package move also needs info in `profiles/updates`: https://devmanual.gentoo.org/ebuild-maintenance/package-moves/index.html#moving-or-renaming-a-package
> 

still confusing to me since slotmoves should now go without it. but done anyway.

**Comment by fordfrog** [comment]
i'm still thinking what would be the correct way to change slot for a java package. maybe some kind of soft blocking the old slot... 🤔

**Comment by vaukai** [comment]
> i'm still thinking what would be the correct way to change slot for a java package. maybe some kind of soft blocking the old slot... 🤔

we could try that with `snappy-java`

File: profiles/updates/3Q-2024 [hunk 1]
```diff
@@ -2,3 +2,4 @@ move sci-libs/rtree dev-python/rtree
 move dev-python/mapbox_earcut dev-python/mapbox-earcut
 move dev-qt/kdsingleapplication dev-libs/kdsingleapplication
 slotmove dev-util/kdevelop-pg-qt 5 0
+move dev-java/ant-ivy dev-java/apache-ivy
```
Comment (fordfrog) [inline profiles/updates/3Q-2024:5]: as the recent case we had, the move should be dropped

**Review by fordfrog — CHANGES_REQUESTED** [review]

File: profiles/updates/3Q-2024 (hunk 1 above)
Comment (vaukai) [reply profiles/updates/3Q-2024]: > as the recent case we had, the move should be dropped

done.

**Comment by vaukai** [comment]
Too low priority, closing.

**Pull request closed without merging**